src/annotator/config/index.js

```diff
@@ -1,53 +1,240 @@
 import settingsFrom from './settings';
 import { toBoolean } from '../../shared/type-coercions';
 
+/**
+ * @typedef ConfigDefinition
+ * @prop {boolean} allowInBrowserExt - Allow this name to be available in the browser extension
+ * @prop {any} [defaultValue] - Sets a default if `valueFn` returns undefined
```

Comment:
Then it will be whatever the settings define it to be unless there is a custom coerce method. 
note: that would likely change once we actually had real validation.  A setting would need to pass a test to move on in that case.